src/main/resources/com/google/api/codegen/java/settings.snip

```diff
@@ -316,33 +316,35 @@
           public String getBundlePartitionKey({@desc.requestTypeName} request) {
             return {@partitionKeyCode(desc)};
           }
-
+          
           @@Override
-          public {@desc.requestTypeName} mergeRequests(Collection<{@desc.requestTypeName}> requests) {
-            {@desc.requestTypeName} firstRequest = requests.iterator().next();
-
-            {@desc.bundledFieldTypeName} elements = new ArrayList<>();
-            for ({@desc.requestTypeName} request : requests) {
-              elements.addAll(request.{@desc.bundledFieldGetFunction}());
-            }
-
-            {@desc.requestTypeName} bundleRequest =
-                {@desc.requestTypeName}.newBuilder()
-                    {@fieldCopyCalls(desc.discriminatorFieldCopies)}
-                    .{@desc.bundledFieldSetFunction}(elements)
-                    .build();
-            return bundleRequest;
+          public BundlingDescriptor.RequestBuilder<{@desc.requestTypeName}> getRequestBuilder() {
```

Comment:
I guess this is more of a comment on the GAX change, but I'd prefer not to use inner classes when code outside of the package has to extend/implement it.